### PR TITLE
Adding Well Known Text Conversions

### PR DESCRIPTION
**Pull request by JCOR11599**
These are functions that allow a user to create spatial types using the Well Known Text (WKT) format.
Functions:
- `point_from_text`
- `line_from_text`
- `poly_from_text`
- `geom_from_text`
- `as_text`
- `as_wkt`

### CODE REVIEW

File: sql/expression/function/line_from_text.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	}
+
+	// Separate by comma
+	pointStrs := strings.Split(s, ",")
+
+	// Parse each point string
+	var points = make([]sql.Point, len(pointStrs))
+	for i, ps := range pointStrs {
+		// Remove leading and trailing whitespace
+		ps = strings.TrimSpace(ps)
```
Comment (andrew-wm-arthur) [inline sql/expression/function/line_from_text.go:84]: also needs to consider surround parenthesis:
https://dev.mysql.com/doc/refman/8.0/en/gis-wkt-functions.html

**Review by andrew-wm-arthur — COMMENTED** [review]
Needs to handle both standard and non-standard WKT format, with tests

File: sql/expression/function/line_from_text.go (hunk 1 above)
Comment (JCOR11599) [reply sql/expression/function/line_from_text.go:84]: I'm not sure what the non-standard wkt format is?
Also, I think surround parenthesis is only for MultiPoint, which I don't have yet.
`select st_aswkt(st_geomfromtext("linestring((0 0), (1 1))"));` throws an error in MySQL

**Review by andrew-wm-arthur — APPROVED** [review]
LGTM!

**Pull request merged**